software_reseller/data/ir_actions_act_window.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="software_reseller.all_licenses_ce314e8b-2439-473a-b7a7-493af7707108" model="ir.actions.act_window" context="{'studio': True}">
+        <field name="domain">[('project_id', 'ilike','%licenses%')]</field>
+        <field name="help">&lt;p class="o_view_nocontent_smiling_face"&gt;
```

Comment:
this help field should be adapted to explain what is supposed to be displayed here, something like 'This list contains tasks from project having 'licences' in its name